src/gwt/src/org/rstudio/studio/client/application/StudioClientApplicationConstants_fr.properties

```
@@ -93,7 +93,7 @@ applicationUpdatedCaption=Application mise à jour
 applicationUpdatedMessage=Une version mise à jour de RStudio est disponible. Votre navigateur va maintenant être rafraîchi avec la nouvelle version. Tous les travaux et données en cours seront préservés pendant la mise à jour.
 warningBarText=Barre d''alerte
 rSessionErrorCaption=Erreur de session R
-previousRSessionsMessage=La session R précédente s''est terminée de manière anormale en raison d''un crash inattendu.\n\nVous avez peut-être perdu des données de l''espace de travail à la suite de ce crash.
+previousRSessionsMessage=La session R précédente s''est terminée de manière anormale en raison d''un crash inattendu.\n\nVous avez peut-être perdu des données de l''espace de travail à la suite de ce crash.\n\nRStudio n''a peut-être pas restauré le projet précédemment actif par précaution. Vous pouvez y revenir via le menu Projets. 
```

Minor order of words. I would have write:
```suggestion
previousRSessionsMessage=La session R précédente s''est terminée de manière anormale en raison d''un crash inattendu.\n\nVous avez peut-être perdu des données de l''espace de travail à la suite de ce crash.\n\nPar précaution, RStudio n''a peut-être pas restauré le projet précédemment actif. Vous pouvez y revenir via le menu Projets. 
```